src/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -212,7 +208,7 @@ const SearchAndFilter = ({
         })}
         <SearchBox
           autocomplete="off"
-          externallyControlled={externallyControlled}
+          externallyControlled={true}
```

Comment:
Small thing, but the convention for `true` props, is just to list the prop, e.g.

`<Foo aTrueProp />`